### PR TITLE
docs(api): explain usage of `WEBPACK_PACKAGE` env var

### PR DESCRIPTION
**Pull request by snitin315**
Add example usage for the `WEBPACK_PACKAGE` environment variable.

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -531,6 +531,31 @@ NODE_OPTIONS="--max-old-space-size=4096 -r /path/to/preload/file.js" webpack
...
+### WEBPACK_PACKAGE
+
+Use a custom webpack version in CLI. Considering the following entries in your `package.json`:
+
+```json
+{
+  //...
+  "webpack": "^4.0.0",
+  "webpack-5": "npm:webpack@^5.32.0",
+  "webpack-cli": "^4.5.0"
```
Comment (snitin315) [inline src/content/api/cli.md:543]:
```suggestion
  "webpack-cli": "^4.6.0"
```

File: src/content/api/cli.md [hunk 2]
```diff
@@ -531,6 +531,31 @@ NODE_OPTIONS="--max-old-space-size=4096 -r /path/to/preload/file.js" webpack
 WEBPACK_CLI_FORCE_LOAD_ESM_CONFIG=true npx webpack --config ./webpack.config.esm
 ```
 
+### WEBPACK_PACKAGE
+
+Use a custom webpack version in CLI. Considering the following entries in your `package.json`:
```
Comment (chenxsan) [inline src/content/api/cli.md:536]:
```suggestion
Use a custom webpack version in CLI. Considering the following content in your `package.json`:
```
Avoid `entries` as it means something else in webpack.

File: src/content/api/cli.md [hunk 3]
```diff
@@ -531,6 +531,31 @@ NODE_OPTIONS="--max-old-space-size=4096 -r /path/to/preload/file.js" webpack
...
+
+```json
+{
+  //...
```
Comment (chenxsan) [inline src/content/api/cli.md:540]:
```suggestion
```
`json` doesn't allow `// ...`.

File: src/content/api/cli.md [hunk 4]
```diff
@@ -531,6 +531,31 @@ NODE_OPTIONS="--max-old-space-size=4096 -r /path/to/preload/file.js" webpack
 WEBPACK_CLI_FORCE_LOAD_ESM_CONFIG=true npx webpack --config ./webpack.config.esm
 ```
 
+### WEBPACK_PACKAGE
```
Comment (chenxsan) [inline src/content/api/cli.md:534]: Add a link to this heading from the above table.

**Review by chenxsan — APPROVED** [review]

**Pull request merged**